bench/fixture.ts

```diff
@@ -334,4 +336,62 @@ export class BenchFixture {
 
     return await transaction.wait();
   }
+
+  public async settleOrder(): Promise<ContractReceipt> {
+    const {
+      deployment: { settlement },
+      domainSeparator,
+      solver,
+      traders: [trader],
+      uniswapPair,
+      uniswapTokens: [sellToken, buyToken],
+    } = this;
+
+    const sellAmount = ethers.utils.parseEther("1.0");
+    const buyAmount = ethers.utils.parseEther("0.9");
+    const feeAmount = sellAmount.div(1000);
+
+    const order = {
+      sellToken: sellToken.address,
+      buyToken: buyToken.address,
+      sellAmount,
+      buyAmount,
+      validTo: 0xffffffff,
+      appData: this.nonce,
+      feeAmount,
+      kind: OrderKind.SELL,
+      partiallyFillable: false,
+    };
+    const signature = await signOrder(
+      domainSeparator,
+      order,
+      trader,
+      SigningScheme.EIP712,
+    );
+    const transfer = {
+      target: uniswapPair.address,
+      amount: order.sellAmount,
+    };
+    const interaction = {
+      target: uniswapPair.address,
+      value: 0,
+      callData: uniswapPair.interface.encodeFunctionData("swap", [
+        0,
+        buyAmount,
```

Comment:
In principle this should depend on which is token 0 and token 1, right? This could actually be relevant to the bench in terms of storage slots used.

Reply:
I'm not sure I follow.

Reply:
All good! For some reason I though we needed to call Uniswap more than once in the bench, in which case using always the same output token might be an issue.